### PR TITLE
pairwise indices with overlaps

### PR DESCRIPTION
**Pull request by slobodan-ilic**

### CODE REVIEW

File: src/cr/cube/cubepart.py [hunk 1]
```diff
@@ -614,6 +614,15 @@ def pairwise_indices(self):
...
         """
+        # If overlaps are defined, calculate significance based on them
+        if (
+            self.dimension_types[-1] == DT.MR
+            and self._cube.overlaps is not None
+            and self._cube.valid_overlaps is not None
+        ):
+            return self._assembler.pairwise_indices(self._alpha, self._only_larger)
+
+        # If overlaps are not defined, default to the legacy way of calculating sig
```
Comment (ernestoarbitrio) [inline src/cr/cube/cubepart.py:625]: I think that this change is needed also in the `pairwise_indices_alt` to enable the 2 level (aBCdeF) pairwise sig letters

File: tests/integration/test_pairwise_significance.py [hunk 2]
```diff
@@ -571,6 +571,19 @@ def test_pairwise_cat_x_mr_gender_x_all_pets_owned(self):
...
+        assert slice_.column_percentages.tolist() == [
+            [75.0, 20.0, 60.0],
+            [25.0, 80.0, 40.0],
+        ]
+
+        assert slice_.pairwise_indices.tolist() == [
+            [(), (0, 2), ()],
+            [(1,), (), (1,)],
+        ]
+
```
Comment (ernestoarbitrio) [inline tests/integration/test_pairwise_significance.py:586]: I'd add a case with the 2 level of significance

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
per comments

File: src/cr/cube/cubepart.py (hunk 1 above)
Comment (slobodan-ilic) [reply src/cr/cube/cubepart.py:625]: done

File: tests/integration/test_pairwise_significance.py (hunk 2 above)
Comment (slobodan-ilic) [reply tests/integration/test_pairwise_significance.py]: done

File: tests/integration/test_pairwise_significance.py [hunk 3]
```diff
@@ -571,6 +571,27 @@ def test_pairwise_cat_x_mr_gender_x_all_pets_owned(self):
...
+        assert slice_.pairwise_indices_alt.tolist() == [
+            [(), (0, 2), ()],
+            [(1,), (), (1,)],
+        ]
```
Comment (ernestoarbitrio) [inline tests/integration/test_pairwise_significance.py:593]: could you use a sig level alt like 0.001 to see if the indices are different? Just to be sure that that method is doing something different

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
1 minor comment than LGTM

File: tests/integration/test_pairwise_significance.py (hunk 3 above)
Comment (slobodan-ilic) [reply tests/integration/test_pairwise_significance.py]: done

**Review by ernestoarbitrio — APPROVED** [review]

**Pull request merged**